fix error in bookTitle function

hamnox opened the pull request:
This error alerted me to the problem

> TypeError: Cannot read property 'content' of undefined
>     at api.sbot.async.get ($HOME/patchbay/node_modules/patch-book/message/html/render/about-subjective.js:42:31)

hamnox committed:
fix error in bookTitle function

This error alerted me to the problem

> TypeError: Cannot read property 'content' of undefined
>     at api.sbot.async.get ($HOME/patchbay/node_modules/patch-book/message/html/render/about-subjective.js:42:31)

arj03 commented:
Hey @hamnox 

Thanks for the fix, I'm not sure its good though. The error says that msg is undefined and sbot.get returns a message object with the value directly because you just gave it a key so no need to send that back. It appears that the problem is that the error is not handled gracefully. Maybe key doesn't exist yet?

Something like this instead should print out what the error is:

```
if (err) return throw(err)
```

hamnox committed:
remove mistake and add error case handler

Thought I didn't need to make it a proper issue report, but the problem wasn't the simple thing I initially thought and then I stopped being able to reproduce the problem partway through the reinvestigation ¯\_(ツ)_/¯

adding an error case handler so it won't try to update using the failed get

arj03 commented:
Thanks for the fix. I'm working on a total rework of patch-book to make it more in line with the "new" way of building apps. It's in this branch: https://github.com/arj03/patchbay-book/tree/scuttle-book.
